src/util.ts

```diff
@@ -1,7 +1,7 @@
-import type { ZodObject, ZodError } from "zod";
+import {ZodError, ZodObject} from "zod";
 
 type TRecord = Record<string, string | number | File | Array<string> | Array<number> | Array<File>>;
-export function castFormData<T extends TRecord>(formData: FormData, schema: ZodObject<any>): T | ZodError {
```

Comment:
Hi, mate. I wonder if there is any benefit in removing the return type of the function?